### PR TITLE
Re-enable special robot headshot/backstab animations

### PR DESCRIPTION
**Pull request by Thestickman391**

### CODE REVIEW

**Comment by wgetJane** [comment]
are these animations specifically made for the robots or do they just use human animations?

**Comment by Thestickman391** [comment]
They are made specifically for the robots, aside from the engineer

**Review by maximsmol — APPROVED** [review]
Changes LGTM. Why was this disabled in the first place though? Are these animations buggy?

**Comment by Thestickman391** [comment]
[Nope](https://www.youtube.com/watch?v=UhvdkCp5of8)

**Comment by Yttrium-tYcLief** [comment]
They were implemented and simply disabled for some reason. I'm going to see about stress-testing these, in case there's an actual valid reason for them to have been disabled.

File: game/shared/tf/tf_player_shared.cpp [hunk 1]
```diff
@@ -12735,12 +12735,6 @@ int CTFPlayerShared::GetSequenceForDeath( CBaseAnimating* pRagdoll, bool bBurnin
 	if ( !pRagdoll )
 		return -1;
 
-	if ( TFGameRules() && TFGameRules()->IsMannVsMachineMode() )
```
Comment (mastercoms) [inline game/shared/tf/tf_player_shared.cpp:12738]: Surround this with #ifdef VALVE_PURE.

**Review by mastercoms — CHANGES_REQUESTED** [review]

**Comment by Thestickman391** [comment]
Done

**Review by mastercoms — APPROVED** [review]

**Pull request merged**